docs/geocoding.md

```diff
@@ -118,7 +116,7 @@ Place results may be biased toward a given longitude and latitude.
 ```python
 
 >>> response = geocoder.forward(
-...     "200 queen street", lon=-66.1, lat=45.3)
+...     "200 queen street", lon=-66.05, lat=45.27)
```

Comment:
The proximity location bias is very sensitive to location precision. Single digit it was returning an address in Australia. Double digit it returned the proper street in Canada. See https://github.com/mapbox/geocoder-feedback/issues/648#issue-182550459